src/TodoListItem.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 
-const TodoListItem = (props) => {
+const TodoListItem = ({ todo }) => {
```

Comment:
Nice job destructuring here but the variable name is a little misleading. We aren't passing the whole todo object. It's just one specific part of the object now.

Reply:
I am confused. If i change the variable name then it won't display the value of the input after submission. It just shows an empty value. Am I missing something here?

Reply:
You would need to change the variable name where you're passing the variable as well.